### PR TITLE
[skip ci] Escape properly from the space

### PR DESCRIPTION
**Pull request by caglar10ur**
This is from @hickeng.

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
lgtm

File: infra/scripts/bash-helpers.sh [hunk 1]
```diff
@@ -91,6 +91,8 @@ addr-from-dockerhost () {
 #    export COMPUTE=cluster/pool
 #    export DATASTORE=datastore1
 #    export IMAGE_STORE=$DATASTORE/image/path
+#    export MAPPED_NETWORKS="--container-network=VM Network:external"
```
Comment (hickeng) [inline infra/scripts/bash-helpers.sh]: You need to add an example of how to supply multiples - they need to be an array or the expansion won't work correctly for CLI processing:
```
MAPPED_NETWORKS=("--container-network=VM Network:external" "--container-network=SomeOtherNet:elsewhere")
export MAPPED_NETWORKS
```

Comment (caglar10ur) [inline infra/scripts/bash-helpers.sh]: Done

**Comment by andrewtchin** [comment]
lgtm

**Comment by fdawg4l** [comment]
ROFL.  Poor guy

**Comment by fdawg4l** [comment]
LGTM

**Comment by caglar10ur** [comment]
♻️

**Pull request merged**